### PR TITLE
DEV-7919 Accruals example

### PR DESCRIPTION
**Pull request by rizwansaeed**

### CODE REVIEW

File: examples/features/core-lusid/Accruals.ipynb [hunk 1]
```diff
@@ -0,0 +1,578 @@
+{
```
Comment (slemasne) [inline examples/features/core-lusid/Accruals.ipynb:1]: <p>Perhaps add an introduction at the start? What fee is accruing? Why is the portfolio paying the fee? Is this a transaction cost from purchase of the stock? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (slemasne) [inline examples/features/core-lusid/Accruals.ipynb:1]: <p>What's a case balance? Should that be "base currency" balance?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (slemasne) [inline examples/features/core-lusid/Accruals.ipynb:1]: <p>Perhaps remove blank cell at the end? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (rizwansaeed) [reply examples/features/core-lusid/Accruals.ipynb:1]: <p>done</p>

Comment (rizwansaeed) [reply examples/features/core-lusid/Accruals.ipynb:1]: <p>done</p>

File: examples/features/core-lusid/Accruals.ipynb [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
```
Comment (slemasne) [inline examples/features/core-lusid/Accruals.ipynb:1]: <p>"from the main balance" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Pull request merged**